examples/using-redux/wrap-with-provider.js

```diff
@@ -3,7 +3,9 @@ import { Provider } from "react-redux"
 
 import createStore from "./src/state/createStore"
 
-const store = createStore()
-
 // eslint-disable-next-line react/display-name,react/prop-types
-export default ({ element }) => <Provider store={store}>{element}</Provider>
+export default ({ element }) => {
+  // Create a fresh store for each SSR page:
```

Comment:
maybe this would work?
```suggestion
  // Instantiating store in `wrapRootElement` handler ensures:
  //  - there is fresh store for each SSR page
  //  - it will be called only once in browser, when React mounts
```